src/vue/status-bar/PortUtilization.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div>
+    <span>{{ $t("statusbar_port_utilization.message") }}</span>
+    <ReadingStat
+      message="statusbar_usage_download"
+      :value="usageDown"
```

Comment:
: is this a typo or a vue function?

Reply:
It's shorthand for `v-bind:value`.

[More about shorthands](https://vuejs.org/v2/guide/syntax.html#Shorthands)